clustered/integration-test/src/test/java/org/ehcache/clustered/sync/PassiveSyncTest.java

```diff
@@ -33,11 +33,16 @@
 import org.terracotta.testing.rules.BasicExternalCluster;
 import org.terracotta.testing.rules.Cluster;
 
+import com.google.code.tempusfugit.temporal.Timeout;
+
 import java.io.File;
 import java.util.Collections;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import static com.google.code.tempusfugit.temporal.Duration.seconds;
+import static com.google.code.tempusfugit.temporal.Timeout.timeout;
+import static com.google.code.tempusfugit.temporal.WaitFor.waitOrTimeout;
```

Comment:
Was added by @chrisdennis a while ago, so decided to use it instead of rolling a similar logic.